app-config.yaml

```diff
@@ -142,14 +142,50 @@ kubernetes:
     type: 'multiTenant'
   clusterLocatorMethods:
     - type: 'config'
-      clusters:
-        # Local development cluster configuration
-        # Override these values in app-config.local.yaml or app-config.production.yaml
-        - url: ${KUBERNETES_API_URL}
-          name: ${KUBERNETES_CLUSTER_NAME}
-          authProvider: 'serviceAccount'
-          skipTLSVerify: true
-          serviceAccountToken: ${KUBERNETES_SERVICE_ACCOUNT_TOKEN}
+      clusters: []
+        # Clusters are defined in app-config.local.yaml
```

Comment:
```suggestion
      clusters:
        # Local development cluster configuration
        # Override these values in app-config.local.yaml or app-config.production.yaml
        - url: ${KUBERNETES_API_URL}
          name: ${KUBERNETES_CLUSTER_NAME}
          authProvider: 'serviceAccount'
          skipTLSVerify: true
          serviceAccountToken: ${KUBERNETES_SERVICE_ACCOUNT_TOKEN}
```

Reply:
Keep the configuration here, even when it might be overwritten. 
Allows to configure cluster via env variables